applications/fishing-map/src/features/workspace/activity/ActivitySection.tsx

```diff
@@ -148,16 +168,46 @@ function ActivitySection(): React.ReactElement {
     []
   )
   const hasVisibleDataviews = dataviews?.some((dataview) => dataview.config?.visible === true)
-  const presenceOptions = useMemo(
-    () => [
-      {
-        id: 'presence' as WorkspacePresenceCategory,
-        label: t('common.presence', 'Fishing presence'),
-      },
-      { id: 'viirs' as WorkspacePresenceCategory, label: t('common.viirs', 'VIIRS') },
-    ],
-    [t]
-  )
+  const fishingOptions = useMemo(() => {
+    const options = fishingDataviews.map((dataview) => {
+      const option = { id: dataview.id, label: dataview.name }
+      if (dataview.id === DEFAULT_FISHING_DATAVIEW_ID) {
+        option.label = t('common.apparentFishing', 'Apparent Fishing Effort')
+      } else {
+        const datasetId = dataview.datasetsConfig?.find(
+          (d) => d.endpoint === EndpointId.FourwingsTiles
+        )?.datasetId
+        option.label = t(
+          `datasets:${removeDatasetVersion(datasetId || '')}.name` as any,
+          dataview.name
+        )
+      }
+
+      return option
+    })
+    return options
+  }, [fishingDataviews, t])
+
+  const presenceOptions = useMemo(() => {
```

Comment:
There's a lot of repetition with `fishingOptions`, might be worth factoring the logic?

Reply:
Oki doki, will be addressed in https://github.com/GlobalFishingWatch/frontend/pull/815 to avoid conflicts as there were changes on that part